### PR TITLE
refactor all templates to have variables in double quotes

### LINKED ISSUE #125: Playbook failed when PVC name is a number
Add double quotes to string fields on templates to avoid errors creating PVCs on the destination cluster, due numbers on the source PVC names.

**Source PVC:**
```
apiVersion: v1
kind: PersistentVolumeClaim
metadata:
  annotations:
     ... ...
  finalizers:
  - kubernetes.io/pvc-protection
  name: "33"
  namespace: my-namespace
spec:
   ... ...
```

**Error:**
```
Failed to create object: b'{\"kind\":\"Status\",\"apiVersion\":\"v1\",\"metadata\":{},\"status\":\"Failure\",\"message\":\" \\\\\"\\\\\" is invalid: patch: Invalid value: ... ... ...
v1.PersistentVolumeClaim.ObjectMeta: v1.ObjectMeta.Name: ReadString: expects \\\\\" or n, but found 1, error found in #10 byte of ...|],\\\\\"name\\\\\":33,\\\\\"namesp|...,", "reason": "Unprocessable Entity", "status": 422}
```

**Fix:**
On file: https://github.com/konveyor/pvc-migrate/blob/master/2_pvc_destination_gen/templates/pvc.yml.j2 - add double quotes to name.
```
metadata:
  finalizers:
  - kubernetes.io/pvc-protection
  name: "{{ item.pvc_name }}"     <<-- HERE
  namespace: {{ item.pvc_namespace }}
spec:
```

### PR DESCRIPTION
**Pull request by alaypatel07**
attempts to fix #125, pending testing

### CODE REVIEW

File: 3_run_rsync/templates/pod.yml.j2 [hunk 1]
```diff
@@ -1,48 +1,48 @@
...
-        memory: {{ transfer_pod_mem_requests }}
+        cpu: "{{ transfer_pod_cpu_requests }}"
+        memory: "{{ transfer_pod_mem_requests }}"
     imagePullPolicy: Always
     command:
       - bash
       - -c
-      - "echo {{ mig_dest_ssh_user }}:$(echo $rsyncd_pass | base64 -d) > /etc/rsyncd.secrets && chmod 0600 /etc/rsyncd.secrets && /usr/bin/rsync --daemon --no-detach --port {{ rsyncd_port }} -vvv"
-    image: {{ transfer_pod_image }}
+      - "echo "{{ mig_dest_ssh_user }}":$(echo $rsyncd_pass | base64 -d) > /etc/rsyncd.secrets && chmod 0600 /etc/rsyncd.secrets && /usr/bin/rsync --daemon --no-detach --port "{{ rsyncd_port }}" -vvv"
```
Comment (alaypatel07) [inline 3_run_rsync/templates/pod.yml.j2]: this needs to be fixed

File: 3_run_rsync/templates/rsyncd.yml.j2 [hunk 2]
```diff
@@ -13,22 +13,22 @@ data:
...
     gid = root
 
 
 {% for pvc in pvcs %}
-    [{{ pvc.pvc_vol_safe_name }}]
-        comment = archive for {{ pvc.pvc_vol_safe_name }}
-        path = /mnt/{{ pvc_namespace }}/{{ pvc.pvc_vol_safe_name }}
+    ["{{ pvc.pvc_vol_safe_name }}"]
+        comment = archive for "{{ pvc.pvc_vol_safe_name }}"
+        path = /mnt/"{ pvc_namespace }}"/"{ pvc.pvc_vol_safe_name }}"
```
Comment (pranavgaikwad) [inline 3_run_rsync/templates/rsyncd.yml.j2]: Full quotes here: `"/mnt/{ pvc_namespace }}/{ pvc.pvc_vol_safe_name }}"`?

**Review by pranavgaikwad — CHANGES_REQUESTED** [review]
Overall, good. Question about one change in the `rsyncd` j2 template

**Review by pranavgaikwad — APPROVED** [review]
ACK

**Pull request merged**